### ISSUE TITLE
No Ninjas

### ISSUE THREAD

**Issue by Astewarthwi** [ErAz7/Ninja#7]
I installed Ninja on my spare laptop, and Jonin on my main laptop.  

Ninja config - Port 3707 and Host 192.168.1.102
Jonin Config - Port 3707

When I run Jonin on my main laptop, I get No Ninjas error.

On the Ninja laptop, I tried telnet 192.168.1.102 3707 but it won't connect, so it seems that Ninja isn't actually opening any port.  The service is running in the windows 10 service manager, and I don't see anything in the logs to indicate any errors.

I do also have port forwarding setup in my router, but the fact that I can't telnet to the port on the local machine seems to indicate it's not a router issue, but something on the machine itself.

Any thoughts what might be causing it?

**Comment by ErAz7** [comment]
well actually Ninja does not open any port, Jonin does that and Ninja starts a polling until Jonin answers and it establishes the connection, thus the `host` and `port` config in Ninja should point to `Jonin's IP address and port`

about port forwarding, you dont need it if both machines are on same network

hope this helps, lemme know if any other issues

**Comment by Astewarthwi** [comment]
Hey, Thanks for getting back to me so quick!   

ahhhh ok, so I had it backwards.   I changed the host in the Ninja config to point to the IP address of the laptop that Jonin runs on.  I restarted the ninja service.   On the laptop with Jonin, I ran "list" but it still says no Ninjas.   From the machine that Ninja is installed on, I can telnet to the IP address and port for the Jonin machine, so seems to see it on the network, but still not getting any Ninjas when I type list in Jonin
